CONTRIBUTING.rst

```diff
@@ -46,11 +46,11 @@ Or Ubuntu with:
 
 .. code:: bash
 
-    wget -qO- https://packages.microsoft.com/keys/microsoft.asc | sudo apt-key add -
```

Comment:
Maybe keep this sudo — this one will definitely be needed, and properly adding `sudo` is more complex here than everywhere else since it's not just "prefix the line with `sudo`".

Either that or maybe split this command into two steps:
```
wget -qO- https://packages.microsoft.com/keys/microsoft.asc > /tmp/microsoft.asc
apt-key add /tmp/microsoft.asc
```
where only the second one needs `sudo`.